webpack/InsightsCloudSync/Components/InsightsTable/__tests__/fixtures.js

```diff
@@ -0,0 +1,102 @@
+import Immutable from 'seamless-immutable';
+import { noop } from 'patternfly-react';
+
+const hits = Immutable([
```

Comment:
Can hits only have the needed props? It doesn't have here `hostname` which is used in the table but has `last_seen`

Reply:
I prefer the API responses to be as accurate as possible